src/components/Eligibility.vue

```diff
@@ -1,88 +1,135 @@
+        <strong>Sorry.</strong> Participants must be at least 18 years of age to register.
+      </div>
+  
+      <!--Field input two-->
+      <div v-if="isUnderage !== null && !isUnderage" class="input-group col-md-4">
+        <label class="form-input-label mr-2 inputLabel"> I live in </label>
+        <input v-model="zipCode"  id="placeField" class="form-control inputLabel" type="number" pattern="\d*" placeholder="enter 5-digit zipcode"></input>
+      </div>
+    </div>
 
-    <!--Field input two-->
-    <span v-if="ageAnswered">
-      <p> I live in </p>
-      <input id="placeField" type="text" v-on:keyup="response"></input>
-    </span>
+    <!--TODO: incorporate error message without immediately prompting the user -->
+    <!--<div v-if="zipCode !== '' && zipCode.length < 5" class="alert alert-danger col-md-4 offset-md-4" role="alert">
+      <strong>Sorry.</strong> Unrecognized zipcode, if there is a mistake please email sagebase.org
+    </div>-->
 
-    <!--Field input three-->
-    <span v-if="holder_one">
-      <br>
-      <p> and I feel </p>
-      <input id="comfortable" type="text" v-on:keyup="response"> </input>
-      <p> using my iPhone </p>
-    </span>
+      <!--Field input three-->
+    <div class="row input-group" v-if="isPlaceAnswered">
+      <label class="col-6 form-input-label inputLabel col-md-2 offset-md-2"> and I feel </label>
+      <!--TODO: Fix allignment for medium screens where specific breakpoint makes un asthetic gap between label
+      and the select field-->    
+      <select class="col-6 custom-select text-md-right col-md-2" id="comfortable" placeholder="please select one" v-model="selectedOptionForPhone">
+        <!--TODO: Fill in with actual values-->
+        <!--TODO: Find a way to wrap the text on mobile devices-->
+        <option disabled value="">Please select</option>
+        <option> comfortable </option>
+        <option> weary </option>
+        <option> uncomfortable </option>
+      </select>       
+      <label class="form-input-label inputLabel col-12 col-md-6"> using my mobile device </label>
+    </div>
 
-    <br>
-    <br>
-    <button v-on:click="clicked" id="submission"> Submit </button>
-  </div>
+   <br>
+   <br>
+   <br>
 
+    <div class="row">
+        <br>
+        <br>
+        <div class="col-md-12">
+          <button v-on:click="clicked" id="next"> Submit </button>
+        </div>
+      </div>
+    </div>
 </template>
 
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script src="https://unpkg.com/lodash@4.13.1/lodash.min.js"></script>
 
 <script>
-  export default {
-    name: 'eligibility',
-    data () {
-      return {
-        ageAnswered: false,
-        placeAnswered: false,
-        holder_one: false, // Might fix later, function response is called twice
-        // for unknown reason, works for now
-        holder_two: false,
-        holder_three: false
+export default {
```

Comment:
Great job cleaning up the public data object and splitting the functionality between computed data and watch functions. Really easy to follow